cluster/mesos/docker/docker-compose.yml

```diff
@@ -148,6 +149,8 @@ scheduler:
   - etcd
   - mesosmaster1
   - apiserver
+  volumes:
+  - ./static-pod.json:/static-pods/static-pod.json
```

Comment:
the scheduler wants a path to a directory that contains pod config files. i'm happy to re-parent /static-pods under /opt.